.vault-config/product-builds-engkeyvault.yaml

```
@@ -138,6 +138,18 @@ secrets:
       organizations: dnceng devdiv microsoft
       scopes: packaging_write 
 
+  # OneLocBuild
+  dn-bot-ceapex-package-r:
+    type: azure-devops-access-token
+    parameters:
+      domainAccountName: dn-bot
+      domainAccountSecret:
+          location: helixkv
+          name: dn-bot-account-redmond
+      name: dn-bot-ceapex-package-r
+      organizations: ceapex
+      scopes: packaging_read
```

```suggestion
      scopes: packaging
```

Read-only scope doesn't use the suffix.